src/main/java/io/github/nucleuspowered/nucleus/modules/admin/commands/BlockZapCommand.java

```diff
@@ -39,14 +53,21 @@
 
     @Override public CommandResult executeCommand(CommandSource src, CommandContext args) throws Exception {
         Location<World> location = args.<Location<World>>getOne(this.locationKey).get();
+        String getBlockZapped = location.getBlockType ().getName ();
+        ItemStack item = ItemStack.builder()
+                .fromBlockState(location.getBlock())
+                .build();
+        Text itemText = item.get(Keys.DISPLAY_NAME).orElse(Text.of(item.getTranslation().get())).toBuilder()
+                .onHover(TextActions.showItem(item.createSnapshot()))
+                .build();
         if (location.getBlockType() == BlockTypes.AIR) {
             throw new ReturnMessageException(Nucleus.getNucleus()
                     .getMessageProvider().getTextMessageWithFormat("command.blockzap.alreadyair", location.getPosition().toString(), location.getExtent().getName()));
         }
 
         if (CauseStackHelper.createFrameWithCausesWithReturn(c -> location.setBlock(BlockTypes.AIR.getDefaultState(), BlockChangeFlags.ALL), src)) {
-            src.sendMessage(Nucleus.getNucleus()
-                    .getMessageProvider().getTextMessageWithFormat("command.blockzap.success", location.getPosition().toString(), location.getExtent().getName()));
+
+            src.sendMessage(Nucleus.getNucleus().getMessageProvider().getTextMessageWithTextFormat("command.blockzap.success", Text.of(location.getPosition().toString()), Text.of(location.getExtent().getName()), itemText));
```

Comment:
This statement is really too long to not be broken up to at least a second line.